libs/sdk-js/src/schema.ts

```diff
@@ -219,3 +219,19 @@ export interface Checkpoint {
   checkpoint_id: Optional<string>;
   checkpoint_map: Optional<Record<string, unknown>>;
 }
+
+export interface ListNamespaceResponse {
+  namespaces: string[][];
+}
+
+export interface SearchItemsResponse {
+  items: Item[];
+}
+
+export interface Item {
+  namespace: string[];
+  key: string;
+  value: Record<string, any>;
+  created_at: string;
+  updated_at: string;
```

Comment:
In JS I added these as `createdAt` and `updatedAt`...

To keep parity we should probably re-map these values after fetching so that they aren't returned in the camel case format

Reply:
cool can do